src/Portman.ts

```diff
@@ -252,9 +252,16 @@ export class Portman {
       // Create oaOutput file if it doesn't exist
       fs.outputFileSync(openApiSpecPath, '', 'utf8')
 
-      await execShellCommand(
-        `npx openapi-format ${openApiSpec} -o ${openApiSpecPath} --yaml --filterFile ${filterFile}`
-      )
+      // await execShellCommand(
```

Comment:
don't leave commented code in place... use it or lose it

Reply:
Removed it